src/rsa/convert_nist_rsa_test_vectors.py

```diff
@@ -197,17 +197,20 @@ def main(fn, test_type, padding_alg):
         e = int(case['e'], 16)
         d = int(case['d'], 16)
 
-        if n.bit_length() // 8 < 2048 // 8:
-            debug("Skipping due to modulus length (too small).", DEBUG)
-            continue
-
         if test_type == 'sign':
+            if n.bit_length() < 2048:
```

Comment:
I think it was intentional that we used `n.bit_length() // 8 < 2048 // 8` instead of `n.bit_length() < 2048` previously, right? we want to test the odd key lengths that are close to being correct. Or am I misunderstanding why this was changed?